tests/apollo/test_skvbc.py

```diff
@@ -75,9 +75,8 @@ async def test_state_transfer(self, bft_network,exchange_keys=True):
         await bft_network.wait_for_state_transfer_to_start()
         await bft_network.wait_for_state_transfer_to_stop(0, stale_node)
         await skvbc.assert_successful_put_get(self)
-        random_replica = random.choice(
-            bft_network.all_replicas(without={0, stale_node}))
-        bft_network.stop_replica(random_replica)
+        bft_network.stop_replicas(bft_network.random_set_of_replicas(
+            bft_network.config.f, without={0, stale_node}))
```

Comment:
This is great, but can you actually move this logic into the `bft.force_quorum_including_replica()` method and use it wherever applicable?